capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -6,28 +6,55 @@ import { act } from "react-dom/test-utils";
 import BookSearchTile from "./components/BookSearchTile.jsx";
 import App from "./App.jsx";
 import {
-  Route, 
-  BrowserRouter as Router, 
+  Route,
+  BrowserRouter as Router,
   Link
-} from 'react-router-dom'; 
+} from 'react-router-dom';
 
 const container = document.createElement("div");
 document.body.appendChild(container);
 const book = {
   title: "Title",
-  author: ["Author"],
+  authors: ["Author"],
   thumbnailLink: "https://http.cat/100.jpg"
 };
+const getUserBookLists = () => {
+  let userBookLists = [{ id: "1", name: "Best Books" },
+  { id: "2", name: "Fantasy" },
+  { id: "3", name: "Sci-fi" }];
+  return userBookLists;
+}
+
+const getUserClubs = () => {
+  let userClubs = [{ id: "1", name: "Cool Kats Book Club" },
+  { id: "2", name: "Sci-fi Fans" }];
+  return userClubs;
+}
+
+const getUserCommunities = () => {
+  let userCommunities = [{ id: "1", name: "Sci-fi Community" },
+  { id: "2", name: "Read-a-lot" }];
+  return userCommunities;
+}
 
 test('renders book search tile test', () => {
-  render(<BookSearchTile title={book.title} author={book.author} thumbnailLink={book.thumbnailLink} />, container);
+  render(<BookSearchTile book={book}
+    userBookLists={getUserBookLists()}
+    userClubs={getUserClubs()}
+    userCommunities={getUserCommunities()} />, container);
 
   const bookThumbnail = document.getElementsByClassName("book-img-med")[0];
   expect(bookThumbnail).toBeInTheDocument();
 
-  const bookTitle = document.getElementsByClassName("book-title")[0];
+  const bookTitle = document.getElementsByTagName("h2")[0];
   expect(bookTitle).toBeInTheDocument();
 
-  const bookAuthor = document.getElementsByClassName("book-author")[0];
+  const bookAuthor = document.getElementsByTagName("p")[0];
```

Comment:
What was the reasoning for changing this to use tag names? Just worried in the case that we add other tags and this breaks. What would you think of giving them ids and selecting with those?

Reply:
I removed the class names/ids for the title and author element because they currently aren't referenced in the CSS or elsewhere in the code - if it's valid to add an id to the element just so that I can test easier I'll definitely implement that. Thoughts @geoffbass ?

Reply:
Discussed offline and I have now added class names to the elements and referred to them by class name in the test!